src/app/utils/BlueprintFunctions.js

```diff
@@ -360,23 +377,6 @@ export function setRandom(ship, m) {
   setPercent(ship, m, Math.random() * 100);
 }
 
-/**
- * Set a modification feature value
- * @param {Object}      ship          The ship for which to perform the modifications
- * @param {Object}      m             The module for which to perform the modifications
- * @param {string}      featureName   The feature being set
- * @param {number}      value         The value being set for the feature
- */
-function _setValue(ship, m, featureName, value) {
-  if (Modifications.modifications[featureName].type == 'percentage') {
-    ship.setModification(m, featureName, value * 10000);
-  } else if (Modifications.modifications[featureName].type == 'numeric') {
-    ship.setModification(m, featureName, value * 100);
-  } else {
-    ship.setModification(m, featureName, value);

```

Comment:
Same as with modifierActions in another example, there is always a numeric value in the respective coriolis-data file. And then, it doesn't make much sense to set the value if it's a string or an object, then why risk and pass the unknown value to the module modification. Maybe it would make sense to log it instead?

Reply:
Note that `type` here is not the data type. This can also be `object`, i.e., for `damagedist`. So the branch needs to be restored!

Reply:
Note that `type` here is not the data type. This can also be `object`, i.e., for `damagedist`. So the branch needs to be restored!

Reply:
@felixlinker I just checked  - this branch was not dropped. This is how this part looks now:
```
if (Modifications.modifications[featureName].type == 'percentage') {
      value = value * 10000;
    } else if (Modifications.modifications[featureName].type == 'numeric') {
      value = value * 100;
    }

    cb(featureName, value);
```
So, it was removed, but the logic remains - the value will be passed as is, this else would just be redundant then.